### ISSUE TITLE
Brush selection update issue and custom image for point

### ISSUE THREAD

**Issue by fisherspoons** [apexcharts/apexcharts.js#1449]
Hi, I ran into several problems, maybe there are already solutions, but I don’t see how to do it yet.

[Example](https://codepen.io/fisherspoon/pen/GRJzOWr)

1.Is it possible for an additional schedule (brash) to update separately the entire period and the selected period on it using **updateOptions**?
For example, I have a period of 1 year, when I click on the button (6 months) on an additional chart, select only the area of ​​the selected period, and do not change the rest.
![Screenshot_2](https://user-images.githubusercontent.com/54633510/77922587-e974b200-72a9-11ea-8b88-30fd4cef7562.png)

2. If it is possible to add a download of your image for a point, because for example I need a point with a shadow, but there is no shadow as a property)

**Comment by fisherspoons** [comment]
And there is another heisenbug that I cannot catch. When in the mobile version, for example, I chose a period, and
 scroll the page with the chart, it can be reset completely to its initial state

**Comment by badhrudeen** [comment]
Hi - can anyone suggest, do we have solution for the above issue.

**Comment by junedchhipa** [comment]
@fisherspoons I confirm the bug, it will be addressed soon.

**Comment by geomatupen** [comment]
Having similar issue. After dragging/moving the selection, chart stops updating dynamically which is perfectly working before dragging/moving the selection. I am trying to update the selection dynamically using updateOptions Method.

**Comment by varun2948** [comment]
i am also facing the same bug as @geomatupen have mentioned is there any solutions for it ?? eagerly awaiting the issue hope developers are on it.

**Comment by geomatupen** [comment]
The issue is solved after upgrading the version to 3.19.2.

**Comment by junedchhipa** [comment]
Closing as the scroll behavior is fixed in v3.19.2 as confirmed by @geomatupen

**Comment by rinogo** [comment]
For what it's worth (and for Google), I believe this is the root cause for what I was seeing in my app:
When a jQuery tooltip was displayed, it would cause some funky behavior in our radialBar chart - almost as if the library were "rerendering" the widget, but incorrectly. I also noticed that resizing the window would trigger the same odd behavior. Drilling down further, I found that even just appending an element to the document body would trigger the same behavior.

Fingers crossed that upgrading to 3.19.2 fixes the problem!
